pkg/prometheus/rules_test.go

```diff
@@ -159,7 +159,7 @@ func shouldSplitUpLargeSmallIntoTwo(t *testing.T) {
 		t.Fatalf("expected no error but got: %v", err)
 	}
 
-	if len(configMaps) != 2 {
+	if len(configMaps) != 3 {
 		t.Fatalf("expected rule files to be split up into two ConfigMaps, but got '%v' instead", len(configMaps))
```

Comment:
Forgot to update the error message. But I fee like this test should be updated to continue testing the code path where configuration exceeds the 3 ConfigMaps